src/command.rs

```diff
@@ -72,6 +72,31 @@ pub extern "C" fn wgpu_command_encoder_copy_texture_to_texture(
         copy_size))
 }
 
+type RenderPassColorAttachmentDescriptor =
+    wgt::RenderPassColorAttachmentDescriptorBase<wgc::id::TextureViewId>;
+type RenderPassDepthStencilAttachmentDescriptor =
+    wgt::RenderPassDepthStencilAttachmentDescriptorBase<wgc::id::TextureViewId>;
+
+#[repr(C)]
+pub struct RenderPassDescriptor<'a> {
+    color_attachments: &'a RenderPassColorAttachmentDescriptor,
+    color_attachments_length: usize,
+    depth_stencil_attachment: Option<&'a RenderPassDepthStencilAttachmentDescriptor>,
+}
+
+impl<'a> Into<wgc::command::RenderPassDescriptor<'a>> for &RenderPassDescriptor<'a> {
```

Comment:
I think it's an antipattern to have `Into` implemented for `&Something`. When I see `xxx.into_yyy()` I expect stuff to be moved out, but with `&Something` it still gets borrowed, effectively implying `xxx.to_yyy()` semantics instead. Let's just make this a method on `RenderPassDescriptor`